module/zcommon/zpool_prop.c

```diff
@@ -381,6 +381,12 @@ vdev_prop_init(void)
 	zprop_register_number(VDEV_PROP_INITIALIZE_ERRORS,
 	    "initialize_errors", 0, PROP_READONLY, ZFS_TYPE_VDEV, "<errors>",
 	    "INITERR", B_FALSE, sfeatures);
+	zprop_register_number(VDEV_PROP_TRIM_ERRORS, "trim_errors", 0,
+	    PROP_READONLY, ZFS_TYPE_VDEV, "<errors>", "TRERR", B_FALSE,
```

Comment:
Wouldn't `TRIMERR` be more consistent with `INITERR` above and `TRIMSUP` below?

Reply:
yes, I'll make the change. Thanks.